examples/llamaindex_examples/fastapi/app.py

```
@@ -0,0 +1,278 @@
+import os
+from typing import List, Optional
```

Ruff (0.8.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)